client/driver/docker_test.go

```diff
@@ -1092,6 +1092,30 @@ func TestDockerDriver_ForcePull(t *testing.T) {
 	}
 }
 
+func TestDockerDriver_ForcePull_RepoDigest(t *testing.T) {
+	if !tu.IsTravis() {
+		t.Parallel()
+	}
+	if !testutil.DockerIsConnected(t) {
+		t.Skip("Docker not connected")
+	}
+
+	task, _, _ := dockerTask(t)
+	task.Config["load"] = ""
+	task.Config["image"] = "library/busybox@sha256:58ac43b2cc92c687a32c8be6278e50a063579655fe3090125dcb2af0ff9e1a64"
+	task.Config["force_pull"] = "true"
+
+	client, handle, cleanup := dockerSetup(t, task)
+	defer cleanup()
+
+	waitForExist(t, client, handle)
+
+	_, err := client.InspectContainer(handle.ContainerID())
```

Comment:
Check that the inspected container's image is whats expected.

Reply:
I can do this, but the remote digest won't match the local digest due to "[reasons](https://github.com/docker/distribution/issues/1662)" so I'll have to just hard-code in another id for the local digest to match against. Not ideal